src/General/Button/Button.tsx

```diff
@@ -16,6 +16,7 @@ const SOLID_BUTTON_THEME_MAP = {
   [ButtonVariant.SOLID_WHITE]: ButtonTheme.WHITE,
   [ButtonVariant.SOLID_BLUE]: ButtonTheme.BLUE,
 };
+type buttonType = 'button' | 'submit' | 'reset';
```

Comment:
Have added a new proptype, with default value as 'button'. Didn't want to directly add type='button' to as the types can be these too?

Reply:
@ritikamotwani can we make our button prop to extend `React.ButtonHTMLAttributes<HTMLButtonElement>`? I think we can make aries button to be able to receive all html button attributes.

Reply:
Yes, then the `type` by default has the types button, submit and reset.
But I still have set the default type as button else it would be submit from what I read.